src/test/regress/expected/sqlsmith_failures.out

```diff
@@ -142,8 +142,9 @@ where (select pg_catalog.array_agg(id) from sqlsmith_failures.countries)
 
 -- cleanup
 DROP SCHEMA sqlsmith_failures CASCADE;
```

Comment:
hmm, why is this test changing? Did we add the coordinator to the metadata somewhere?

Reply:
Yes, we've added in the view_propagation tests.